libcxx/test/std/atomics/types.pass.cpp

```diff
@@ -29,163 +29,173 @@
 #  include <thread>
 #endif
 
```

Comment:
Could you format the test in a separate commit and revert the changes here? It's really hard to see what actually changed.

Reply:
Will do.
I realized I forgot to run in `libcxx`, I had just ran `std`...

Reply:
Done

Reply:
It is also somewhat readable if you hide whitespaces